setup.cfg

```diff
@@ -49,6 +49,8 @@ package_dir =
 # For more information, check out https://semver.org/.
 install_requires =
     importlib-metadata; python_version<"3.8"
+    biocpy
+    pillow
```

Comment:
also singlecellexperiment; i would also specify a min version of these packages for spatial experiment to behave correctly